src/App.js

```diff
@@ -1,4 +1,13 @@
 import React from 'react';
+import {SafeAreaProvider} from 'react-native-safe-area-context';
 import Expensify from './Expensify';
 
-export default () => <Expensify />;
+const App = () => (
+    <SafeAreaProvider>
+        <Expensify />
+    </SafeAreaProvider>
```

Comment:
Is there a performance gain to using Safe=area-context? Took a look through the docs but didn't spot anything

Reply:
I'm not sure. We were already using this and I just moved it up a few levels since it was re-rendering when the `HomePage` re-rendered. It might be kind of cheap to do in any case, but seemed unnecessary.

Basically, I am trying to minimize the amount of stuff that re-renders when there's an update and the safe area provider is one of those things I imagine only does it's work once and never makes sense to re-render. Might also be missing some context on why we are using this vs whatever solution react-native offers.

Reply:
Ah gotcha, makes sense 👍